sccm/core/plan-design/changes/whats-new-in-version-1706.md

```diff
@@ -79,24 +80,24 @@ For more information, see [software update points](/sccm/core/servers/deploy/con
 With this release, we have improved the integration of Configuration Manager and Azure Active Directory (Azure AD).  These improvements streamline how you configure the Azure services you use with Configuration Manager, and help you to manage clients and users who authenticate though Azure AD.
 
 The improved integration makes the following possible:  
-  -   Azure Services Wizard – This Wizard provides a common configuration experience that replaces the individual workflows to set up the following Azure services you use with Configuration Manager.
-      - **Cloud Management**
-        Enable clients to authenticate by using Azure Active Directory (Azure AD). You can also configure Azure AD User Discovery.
-      - **Log Analytics Connector**
-        Connect to Azure Log Analytics and sync collection data.
-      - **Upgrade Readiness**
-        Connect to Upgrade Readiness and view client upgrade-compatibility data.
-      - **Windows Store for Business**
-        Connect to the on-line store for Windows Store for Business and get apps for your organization that you can deploy with Configuration Manager.
+- Azure Services Wizard – This Wizard provides a common configuration experience that replaces the individual workflows to set up the following Azure services you use with Configuration Manager.
+  - **Cloud Management**
+    Enable clients to authenticate by using Azure Active Directory (Azure AD). You can also configure Azure AD User Discovery.
+  - **Log Analytics Connector**
+    Connect to Azure Log Analytics and sync collection data.
+  - **Upgrade Readiness**
+    Connect to Upgrade Readiness and view client upgrade-compatibility data.
+  - **Windows Store for Business**
+    Connect to the on-line store for Windows Store for Business and get apps for your organization that you can deploy with Configuration Manager.
 
 
   This is done by using an [Azure server web app](/azure/app-service/app-service-authentication-overview) to provide the subscription and configuration details that you otherwise enter each time you set up a new Configuration Manager component or service with Azure. For more information, see [Azure Services Wizard](/sccm/core/servers/deploy/configure/azure-services-wizard).
```

Comment:
This wasn't falling under the "Azure Services Wizard" bullet because of the spacing